docs/setup/env.md

```diff
@@ -4,73 +4,73 @@
 
 Below are the general environment variables needed for the eWallet to run smoothly.
 
-- `MIX_ENV`: Environment in which the application is being ran. `prod` for production.
-- `BASE_URL`: The base to use when building URLs.
-- `PORT`: The port that the application listens on.
-- `EWALLET_SECRET_KEY`: Encryption key used to encrypt some data in the database.
-- `LOCAL_LEDGER_SECRET_KEY`: Encryption key used to encrypt some data in the database.
+-   `MIX_ENV`: Environment in which the application is being ran. `prod` for production.
```

Comment:
Weird spacing brought to you by [prettier](https://github.com/prettier/prettier).